modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/NotificationsSettingsFragment.kt

```diff
@@ -169,6 +188,17 @@ class NotificationsSettingsFragment :
 
             true
         }
+
+        dailyRemindersPreference?.setOnPreferenceChangeListener { _, newValue ->
+            val newBool = (newValue as? Boolean) ?: throw IllegalStateException("Invalid value for hide notification on pause preference: $newValue")
+            settings.dailyRemindersNotification.set(newBool, updateModifiedAt = true)
```

Comment:
As you are storing user notification information in the database, should this also be in the database instead of the user preferences?